lisa/features/network_interface.py

```diff
@@ -32,6 +33,10 @@ def attach_nics(
     ) -> None:
         raise NotImplementedError
 
+    def reload_netvsc(self) -> None:
```

Comment:
It's not a shared logic, which can across platforms. What's the reason needed? Does it mean to reload a network interface? If so, add interface here, and move the implementation to Azure.

Reply:
reload_netvsc() intends to unload and load again the netvsc driver. We would need to do it for few tests cases both in hyper-v and azure.

Reply:
The netvsc is microsoft specified. Let's move it to azure firstly. If something need to shared between Azure and Hyper-V, we can move it, when implementing Hyper-V

Reply:
How about call it `reload` or `reload_module`? The netvsc is microsoft specified.

Reply:
oh well if I will have to pass the driver name then I would better just call the modprobe function of reload() in the testsuite itself.
The idea to use it in azure was because it might be needed for other test cases too.

Reply:
In general, what should be reloaded in the network interface? What's reason to reload it? If it's to reload the network modules, The Azure reloads netvsc, but other platform may reload other modules.
It's better to write out of test cases, this may be needed by other test cases.